ERCS/erc-7092.md

```diff
@@ -1,326 +1,156 @@
+    function batchDecreaseAllowance(address[] calldata _spender, uint256[] calldata _amount) external;
+    function batchTransfer(address[] calldata _to, uint256[] calldata _amount, bytes[] calldata _data) external returns(bool);
+    function batchTransferFrom(address[] calldata _from, address[] calldata _to, uint256[] calldata _amount, bytes[] calldata _data) external returns(bool);
+}
+```
 
-    /**
-    * @notice Lower the allowance of `_spender` by `_amount`
-    * @param _spender the address to be authorized by the bondholder
-    * @param _amount amount of bond to remove approval; _amount MUST be a multiple of denomination
-    */
-    function decreaseAllowance(address _spender, uint256 _amount) external returns(bool);
+### Additional bond parameters Interface
 
-    /**
-    * @notice Remove the allowance for `_spender`
-    * @param _spender the address to remove the authorization by from
-    *
-    * OPTIONAL - interfaces and other contracts MUST NOT expect these values to be present. The method is used to improve usability.
-    */
-    function decreaseAllowanceForAll(address _spender) external returns(bool);
+Functions in the `IERC7092ESG` interface are OPTIONAL. This interface MAY be used to improve the standard usability.
 
-    /**
-    * @notice Moves `_amount` bonds to address `_to`
-    * @param _to the address to send the bonds to
-    * @param _amount amount of bond to transfer. _amount MUST be a multiple of denomination
-    * @param _data additional information provided by the token holder
-    */
-    function transfer(address _to, uint256 _amount, bytes calldata _data) external returns(bool);
+- The `currencyOfCoupon` The currency used for coupon payment may be different from the currency used to repay the principal
+- The `couponType` MAY be employed to signify the interest rate that the issuer has committed to paying to investors, which may take various forms such as zero coupon, fixed rate, floating rate, and more.
+- The `couponFrequency` refers to how often the bond pays interest to its bondholders, and is typically expressed in terms of time periods, such as: Annual, Semi-Annual, Quarterly, or Monthly.
+- The `dayCountBasis` is used to calculate the accrued interest on a bond between two coupon payment dates or other specific periods. Some of the day count basis are: Actual/Actual, 30/360, Actual/360, Actual/365, or 30/365
 
-    /**
-    * @notice Moves all bonds to address `_to`
-    * @param _to the address to send the bonds to
-    * @param _data additional information provided by the token holder
-    *
-    * OPTIONAL - interfaces and other contracts MUST NOT expect these values to be present. The method is used to improve usability.
-    */
-    function transferAll(address _to, bytes calldata _data) external returns(bool);
+A detailed description of the functions can be found [here](../assets/erc-7092/IERC7092Batch.sol)
```

Comment:
The specification section should be understandable on its own, without having to refer to the reference implementation. Would you mind duplicating/moving the function descriptions here?

Reply:
> The specification section should be understandable on its own, without having to refer to the reference implementation. Would you mind duplicating/moving the function descriptions here?

Hello @SamWilsn,

All functions that manage the entire balance of an account have been removed for security purposes. Therefore the`transferferAll` function is not part of the standard anymore. This will be update when merging the PR to Review.